lib/utils/date.ts

```diff
@@ -19,7 +19,7 @@ export const formatDate = (date: Date, dateInput = false): string => {
  */
 export const formatDateYYYYMMDD = (d: Date, withTime = false): string => {
   const formatString = withTime ? 'YYYY-MM-DD, hh:mm a' : 'YYYY-MM-DD';
-  return moment.utc(d).format(formatString);
+  return moment(d).format(formatString);
```

Comment:
> Need to test if this change affects other parts of the application (particularly date of birth)

In the interest of time, can we define a separate function to be used in reports and requests table only (e.g. `formatDateYYYYMMDDLocalTimeZone`)? This way we won't affect the other call sites and can speed up testing and push this out ASAP.